_data/sidebars/home_sidebar.yml

```diff
@@ -50,6 +50,8 @@
         url: /scheduler/kubernetes/stork.html
       - title: Deploying using CSI
         url: /scheduler/kubernetes/csi.html
+      - title: Ark Plugin
```

Comment:
Let's drop this under the "How to" branch and call "Use Portworx with Ark". Need to restrict the number of items under the top level Kuberentes index.

Reply:
Why? I think anything that adds integrations needs to be at a higher level, otherwise, no one will know that the integration exists.

Reply:
Because there is no end to it. Unless we have a rule, all our integrations will keep getting added here. Making the index longer makes it difficult for people to find. You can leave this here for now. We will have to revisit the organization on the new docs site.